### PR TITLE
o-tech-docs + next-express

### PR DESCRIPTION
**Pull request by commuterjoy**
http://matt.chadburn.co.uk/drop/radiator.gif

- [ ] Fix all NBT linting errors
- [ ] Put in to next-registry
- [ ] Link from registry to the radiator - Eg, http://next-radiator.ft.com/apps/ft-next-beacon
- [ ] DNS - http://next-radiator.ft.com
- [x] Fix hashed assets (and delete main.css from repo)
- [x] Fix the application dashboard (detail.html)
- [x] Fix the dependencies dashboard
- [x] Add a few words of explanation to each dashboard
- [x] Travis set-up
- [x] Pingdom on __gtg 
- [x] Add to config-vars

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-var express = require('express');
-var app = express();
-var fs = require('fs');
-var errorMiddleware = require('express-errors-handler').middleware;
+var app = module.exports = require('ft-next-express')({ layoutsDir: __dirname + '/../views/layouts' });
 
-var handlebars = require('handlebars');
-
-var pingdom = require('../lib/pingdom');
+//var pingdom = require('../lib/pingdom');
```
Comment (matthew-andrews) [inline server/app.js]: temporarily disabled?

Comment (commuterjoy) [inline server/app.js]: didn't work, so going to delete

File: Makefile [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
 
 test:
-	origami-build-tools verify
+	nbt verify --skip-layout-checks
 
 deploy:
-	next-build-tools configure
-	next-build-tools deploy
+	nbt configure
+	nbt deploy
```
Comment (matthew-andrews) [inline Makefile]: i think if you're using nbt build you should also be able to use deploy-hashed-assets.  it should just fine too.

Comment (commuterjoy) [inline Makefile]: Yep, realised there's loads to do with fixing this app, so wrote a big checklist. Just added hashed assets to it :)

Comment (matthew-andrews) [inline Makefile]: will leave you too it then :)

**Pull request merged**